diff --git a/devassistant/settings.py b/devassistant/settings.py
index 2bf1f0f..f39054f 100644
--- a/devassistant/settings.py
+++ b/devassistant/settings.py
@@ -15,7 +15,7 @@ SUBASSISTANT_N_STRING = 'subassistant_{0}'
 SUBASSISTANT_N_STRING = 'subassistant_{0}'
 DEPS_ONLY_FLAG = '--deps-only'
 CACHE_FILE = os.path.expanduser('~/.devassistant/.cache.yaml')
-DATA_DIRECTORIES = [os.path.join(os.path.dirname(__file__), 'data'),
+DATA_DIRECTORIES = ["_DATADIR_DEVASSISTANT",
                     '/usr/local/share/devassistant',
                     os.path.expanduser('~/.devassistant')]
 ASSISTANT_ROLES=['crt', 'mod', 'prep']
